phpstan.neon

```diff
@@ -27,3 +27,9 @@ parameters:
         - '#Access to an undefined property Rector\\Contract\\PhpParser\\Node\\StmtsAwareInterface\:\:\$stmts#'
 
         - '#PhpParser\\Node\\Stmt\\Expression is not generic#'
+
+        # false positive
+        -
+            path: 'rules/CodeQuality/Rector/Class_/AddCoversClassAttributeRector.php'
+            message: '#Parameter \#2 \$callback of function array_filter expects \(callable\(string\|null\)\: bool\)\|null, Closure\(string\)\: bool given#'
```

Comment:
I don't get this error and can't understand what's wrong with this code:

```php
$partsWithoutTests = array_filter(
    $classNameParts,
    static fn (string $part): bool => ! in_array(strtolower($part), ['test', 'tests'], true),
);
```

Reply:
You may can lookup this discussion 

- https://github.com/phpstan/phpstan/discussions/8911

Reply:
ok, interesting. Thank you for the link